src/util/util_string.h

```diff
@@ -39,5 +40,11 @@ namespace dxvk::str {
     format1(stream, args...);
     return stream.str();
   }
+
+  inline void strlcpy(char* dst, const char* src, size_t count) {
```

Comment:
any particular reason why we're rolling our own implementation here?

Reply:
strlcpy doesn't exist unless you're on BSD